errors.go

```diff
@@ -30,3 +30,6 @@ const ErrRejected = errorType("response rejected")
 
 // ErrUnsupported indicates an operation is not supported by the transport protocol
 const ErrUnsupported = errorType("unsupported")
+
+// ErrAlreadySubscribed indicates a subscription to events exists for the given channel
+const ErrAlreadySubscribed = errorType("already subscribe to events for given channel id")
```

Comment:
```suggestion
const ErrAlreadySubscribed = errorType("already subscribed to events for given channel id")
```

but this isn't used, was it meant to be or should it be deleted? looks like it'd be hard to check with the current API because we can't do equality checking.